lib/pages/home/home_controller.dart

```diff
@@ -67,14 +67,15 @@ class HomeController extends GetxController {
     final getSurveysUseCase = Get.find<GetSurveysUseCase>();
     // ignore: invalid_use_of_protected_member
     if (_surveys.value.isEmpty) {
-      final surveys = await getSurveysUseCase.call("");
+      final surveys = await getSurveysUseCase.call(GetSurveysInput(cursor: ""));
```

Comment:
What does `cursor: ""` mean? 🤔

Reply:
start from empty 😅  `cursor` in our `graphql API` is the key id of the item you wanted to point to

![image](https://user-images.githubusercontent.com/13435717/128832082-af51cd01-72f4-432a-bd47-3f1732a90cb7.png)

Reply:
![image](https://user-images.githubusercontent.com/13435717/128832837-bb6b82f9-e9e7-4cf5-a9f4-e8666227d1aa.png)

here is an example of how I formed the `query`. In our graphql API, pagination doesn't count by params in the request like `page=0`, `page=1&count=10` like in the REST API; instead, you can specify a `cursor` and query the `amount` of result `after` that cursor, if the `amount` is static, then you can always iterate through the last content (pagination).

You can read more at [the official GraphQL documentation]([https://graphql.org/learn/pagination/#complete-connection-model) for more examples